packages/panels/docs/08-users.md

```diff
@@ -264,3 +264,15 @@ public function panel(Panel $panel): Panel
         ->authPasswordBroker('users');
 }
 ```
+
+## Setting up guest access to a panel
+
+By default, Filament expects to work with authenticated users only. To allow guests to access a panel, you need to avoid using components which expect a signed-in user (such as profiles, avatars), and remove the built-in Authentication middleware:
+
+- Remove the default `Authenticate::class` from the `authMiddleware()` array in the panel configuration.
+- Remove `->login()` and any other [authentication features](#authentication-features) from the panel.
+- Remove the default `AccountWidget` from the `widgets()` array, because reads the current user's data.
```

Comment:
@danharrin should "because reads the current" have the word "it" inserted? ie: "because it..."

Reply:
sorry, fixing